### PR TITLE
document minus operation in Readme.md

### PR DESCRIPTION
**Pull request by MalloZup**
I just documented the minus operation in Readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,6 +146,13 @@ date-time:
 => #<DateTime 1986-12-05T00:00:00.000Z>
 ```
 
+You can perform also `minus` operations
+```clj
+(t/minus (t/date-time 1986 10 14 6) (t/hours 2))))
+#object[org.joda.time.DateTime 0x7daafcd "1986-10-14T04:00:00.000Z"]
```
Comment (seancorfield) [inline README.md]: Given that we show the `#<DateTime 1986-..Z>` format everywhere else in the README, this should probably follow suit for consistency.

Comment (MalloZup) [reply README.md]: @seancorfield  thx for review. yep right. 

After discussion in slack, if the `clj-time` lib is somehow in a archivable state , do you think make sense to continue this PR? 

thx!

Comment (MalloZup) [reply README.md]: I don't have any pb to close  it or continue it, as you wish

Comment (seancorfield) [reply README.md]: If you're happy to update the PR, I'll merge it. The library is "sort of deprecated" but there's no harm in improving the documentation for those folks who are still using it.

**Comment by MalloZup** [comment]
should be ok now! thx :waning_crescent_moon: :dog2:

**Comment by seancorfield** [comment]
Thank you!

**Comment by MalloZup** [comment]
thx @seancorfield ! for such small PR , was pleasure

**Pull request merged**